src/styles/components/_select.scss

```diff
@@ -0,0 +1,60 @@
+$icon-size-sm: space(lg);
```

Comment:
thoughts on leveling this up to a util var since we're defining and using this in primary & secondary navs too